perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -658,7 +658,15 @@ sub can_display_external_id {
 
 sub duration_string {
     my ( $problem, $c ) = @_;
-    my $body = $problem->body( $c );
+    my $body;
+    if ( $c->cobrand->can('show_link_to_council_cobrand') &&
+         $c->cobrand->show_link_to_council_cobrand($c, $problem) ) {
+        my $base_url = $c->cobrand->get_body_handler_for_problem($problem)->base_url;
```

Comment:
If the creation of the link string is moved into show_link_to_council_cobrand, then we don't need get_body_handler_for_problem in Default.pm, which is probably a bit tidier as it's all very specific at the moment.